.agents/skills/cudf-ci-repro/scripts/parse-job-url.py

```
@@ -0,0 +1,54 @@
+# SPDX-FileCopyrightText: Copyright (c) 2026, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Ruff (0.15.12)

[warning] The file is executable but no shebang is present

(EXE002)